update-solana-dependencies.sh

```diff
@@ -38,5 +38,5 @@ crates=(
 
 set -x
 for crate in "${crates[@]}"; do
-  sed -i -e "s#\(${crate} = \"\)\(=\?\).*\(\"\)#\1\2$solana_ver\3#g" "${tomls[@]}"
+  sed -i'' -e "s#\(${crate} = \"\)\(=\?\).*\(\"\)#\1\2$solana_ver\3#g" "${tomls[@]}"
```

Comment:
accidental change?

Reply:
Not really, see this conversation on Discord: https://discord.com/channels/428295358100013066/517163444747894795/832093850117865472

Reply:
Ah got it, thanks :)